src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Win32.cs

```diff
@@ -7,8 +7,36 @@
+    /// You can change the value of any <xref:System.Diagnostics.ProcessStartInfo> property up to the time that the process starts. After you start the process, changing these values has no effect.
+    /// > [!NOTE]
+    /// >  This class contains a link demand at the class level that applies to all members. A <xref:System.Security.SecurityException> is thrown when the immediate caller does not have full-trust permission. For details about security demands, see [Link Demands](/dotnet/framework/misc/link-demands).
+    /// ## Examples
+    /// The following code example demonstrates how to use the <xref:System.Diagnostics.ProcessStartInfo> class to start Internet Explorer, providing the destination URLs as <xref:System.Diagnostics.ProcessStartInfo> arguments.
+    /// [!code-cpp[Process.Start_static#1](~/samples/snippets/cpp/VS_Snippets_CLR/Process.Start_static/CPP/processstartstatic.cpp)]
+    /// [!code-csharp[Process.Start_static#1](~/samples/snippets/csharp/VS_Snippets_CLR/Process.Start_static/CS/processstartstatic.cs)]
+    /// [!code-vb[Process.Start_static#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/Process.Start_static/VB/processstartstatic.vb)]
+    /// ]]></format></remarks>
+    /// <altmember cref="System.Diagnostics.Process"/>
     public sealed partial class ProcessStartInfo
     {
+        /// <summary>Gets the set of verbs associated with the type of file specified by the <see cref="System.Diagnostics.ProcessStartInfo.FileName" /> property.</summary>
+        /// <value>The actions that the system can apply to the file indicated by the <see cref="System.Diagnostics.ProcessStartInfo.FileName" /> property.</value>
+        /// <remarks><format type="text/markdown"><![CDATA[
+        /// The <xref:System.Diagnostics.ProcessStartInfo.Verbs%2A> property enables you to determine the verbs that can be used with the file specified by the <xref:System.Diagnostics.ProcessStartInfo.FileName%2A> property. You can set the <xref:System.Diagnostics.ProcessStartInfo.Verb%2A> property to the value of any verb in the set. Examples of verbs are "Edit", "Open", "OpenAsReadOnly", "Print", and "Printto".
+        /// When you use the <xref:System.Diagnostics.ProcessStartInfo.Verbs%2A> property, you must include the file name extension when you set the value of the <xref:System.Diagnostics.ProcessStartInfo.FileName%2A> property. The file name extension determines the set of possible verbs.
+        /// ## Examples
```

Comment:
Same question as below -- is there/will there be any kind of tooling validatoin of the markdown in these CDATA blocks?

Reply:
Since it's being backported without changes from MS Docs, it has already been validated there.

We do not have plans to have markdown sections validated on the source side. Do you have strong opinions on getting validation? This would require opening a bug in the csharplang repo to request this feature, right?

Reply:
No I don't have a strong opinion. I doubt the compiler is ever going to validate MD/XML mixtures. I was more wondering whether we are setting ourselves up for success by mixing two markdown languages together and whether there is a path out of that. Clearly the api docs repo chose to do that but we have the luxury of time to be sure we have the right model.